packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { Image } from 'react-native';
+import { Formik } from 'formik';
+import {
+  ModalView,
+  Switch,
+  TextInput,
+  Button,
+  styled,
+  FlexedView,
+  PaddedView,
+} from '@apollosproject/ui-kit';
+import AddPrayerHeader from '../AddPrayerHeader';
+
+const StyledButton = styled(() => ({
+  borderRadius: 0,
+}))(Button);
+
+const BottomView = styled(() => ({
+  justifyContent: 'flex-end',
+}))(FlexedView);
+
+const SwitchContainer = styled(() => ({
+  width: '60%',
+  alignSelf: 'center',
+}))(PaddedView);
+
+const AddPrayerForm = memo(({ imgSrc, btnLabel, ...otherProps }) => (
```

Comment:
Is this form setup to update the `imgSrc` when the `<Switch/>` component is toggled? These can exist as pure components but we'll need to make sure that the parent component handles the passing down of data on toggle.

Reply:
Not sure I understand...the `imgSrc` prop is for the avatar and that won't change based on the switch. I could make the prop name more explicit? Or maybe that's not what you're getting at.

Reply:
If I had to guess I bet what Isaac is suggesting is that if the anonymous switch is on, then the image should change to something that isn't the person's profile image. 

![Screenshot 2019-04-22 14 46 58](https://user-images.githubusercontent.com/3229463/56518470-8ca34000-650d-11e9-890f-f8b49a1f8e24.png)